src/Model/Export/Catalog/Entity/ProductVariation.php

```diff
@@ -11,17 +11,21 @@
 class ProductVariation implements ExportEntityInterface
 {
     /** @var Product */
-    private $product;
+    protected $product;
```

Comment:
Since we have accessor methods, I wouldn't touch the visibility of the members. We have to maintain everything we open for extension, so the less the better

Reply:
I think I mainly intended to streamline the scope of the properties with SimpleDataProvider.php that the other product types are being extended from. However, you're right of course and I don't have a use case for extending the ProductVariation.